HousingSearchApi.Tests/V1/E2ETests/Steps/ResponseModels/TransactionDTO.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+
+namespace HousingSearchApi.Tests.V1.E2ETests.Steps.ResponseModels
+{
+    public class TransactionDTO
+    {
+        public Guid Id { get; set; }
+
+        public Guid TargetId { get; set; }
+
+        public Hackney.Shared.HousingSearch.Domain.Transactions.TargetType TargetType { get; set; }
+
+        public short PeriodNo { get; set; }
+
+        public short FinancialYear { get; set; }
+
+        public short FinancialMonth { get; set; }
+
+        public string TransactionSource { get; set; }
+
+        public Hackney.Shared.HousingSearch.Domain.Transactions.TransactionType TransactionType { get; set; }
+
+        public DateTime TransactionDate { get; set; }
+
+        public PersonDTO Person { get; set; }
```

Comment:
Hey Hanna, can you please confirm with me what the use case behind the person data for this endpoint is?